tests/foreman/cli/test_remoteexecution.py

```diff
@@ -344,7 +345,10 @@ def test_positive_run_job_effective_user(
                 'organization-id': module_org.id,
             }
         )
-        assert 'Permission denied' in out
+        if not (
+            rex_contenthost.os_distribution_version.split(".")[0] == '7' and is_open('SAT-30898')
+        ):
+            assert 'Permission denied' in out
```

Comment:
Instead of fetching `os_distribution_version` from _broker_args, its preferred to read it from the installed OS
```suggestion
        if not rex_contenthost.os_version.major == 7 and is_open('SAT-30898'):
            assert 'Permission denied' in out
```